src/main/openapi/ds-storage-openapi_v1.yaml

```diff
@@ -588,3 +591,59 @@ components:
           type: string
           description: 'Self diagnosed health'
           example: 'ok'
+
+  examples:
+    PostDSRecord:
+      value:
+        id: 'ds.radio:oai:du:c124a602-7fe6-470e-a484-c771db44aee1'
+        origin: 'ds.radio'
+        recordType: 'DELIVERABLEUNIT'
+        data:
+          <xip:DeliverableUnit>
+          <Metadata>
+          <PBCoreDescriptionDocument:PBCoreDescriptionDocument>
+          <exampleTitle>Radioavisen</exampleTitle>
+          <exampleDescription>This is an example description of a news broadcast.</exampleDescription>
+          </PBCoreDescriptionDocument:PBCoreDescriptionDocument>
+          </Metadata>
+          </<xip:DeliverableUnit>>
+
+    GetDSRecordWithChildren:
+      value:
+        id: "ds.radio:oai:du:c124a602-7fe6-470e-a484-c771db44aee1"
+        orgid: "null"
+        id_error: false
+        origin: "ds.radio"
+        recordType: "DELIVERABLEUNIT"
+        deleted: false
+        data: |
```

Comment:
why the pipe '|' character ?

Reply:
It's not needed here, but its used to preserve the format of the following lines directly as written